apps/src/studio/blocks.js

```diff
@@ -965,14 +943,28 @@ exports.install = function(blockly, blockInstallOptions) {
       dropdown.setValue(this.VALUES[1][1]); // default to top-left
       this.setHSV(184, 1.0, 0.74);
       if (spriteCount > 1) {
-        this.appendDummyInput().appendTitle(
-          spriteNumberTextDropdown(msg.setSpriteN),
-          'SPRITE'
+        this.interpolateMsg(
+          msg.setSpriteIndex(),
+          () => {
+            this.appendDummyInput().appendTitle(
+              spriteNumberTextDropdown(msg.setSpriteN),
```

Comment:
We don't want to use the `setSpriteN` string in this case, since it includes the string "set actor".

What we want is to use the string `"set actor %1 to the %2 position"` and have the %1 be replaced by a dropdown consisting of just sprite numbers

Reply:
@Hamms, I modified the setSpriteIndex() to "%1 to the %2 position" to prevent "set actor" from being duplicated.  The other way I attempted to pass in the string "set actor %1 to the %2 position" was the implementation I referenced above.  Not sure how to get this to work.  Thoughts

Reply:
Right, but the problem with that is it doesn't address the reason why we started this project in the first place, which is that sentences being broken up between strings is difficult for the translators.

The problem with the implementation you referenced above is that it uses a textinput; the block currently uses a dropdown, not a textinput, and we want to keep it that way.

What we want is something like

```js
this.interpolateMsg(
  msg.setSpritePosition(),
  () => {
    this.appendDummyInput().appendTitle(spriteNumberTextDropdown("{spriteIndex}"), 'SPRITE')
  },
  () => {
    this.appendDummyInput().appendTitle(dropdown, 'VALUE');
  }
)
```